src/database/entities/Photo.ts

```diff
@@ -38,7 +39,11 @@ export class Photo extends BaseEntity {
   @Column({ type: 'varchar', nullable: true })
   credits: string | null;
 
-  @Column({ type: 'tinyint', nullable: true })
+  @Column({
+    type: 'tinyint',
+    nullable: true,
+    transformer: transformTinyIntOrNull,
```

Comment:
What's the benefit of using `transformTinyIntOrNull`?

Reply:
I merged this before I documented this. I've included a comment to explain why we need this in #106.